doc/dev/index.rst

```diff
@@ -1201,7 +1201,7 @@ ceph-qa-suite`_ tool, you will need to install that first. It is designed
 to be installed via Docker, so if you don't have Docker running on your
 development machine, take care of that first. The Docker project has a good
 tutorial called `Get Started with Docker Engine for Linux
-<https://docs.docker.com/linux/>`_ if you unsure how to proceed.
+<https://docs.docker.com/get-started/>`_ if you unsure how to proceed.
```

Comment:
The Docker organization removed this tutorial, and I can't find any mirror. Maybe we could delete this sentence or suggest [other tutorial](https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-getting-started)